November Challenge/Minimum Cost to Move Chips to The Same Position

```diff
@@ -0,0 +1,11 @@
+class Solution {
+public:
+    int minCostToMoveChips(vector<int>& position) {
+        int odd=0, even=0;
+        for(int pos : position){
+            if((pos & 1 )==0) odd++;
```

Comment:
Hey Nithin,
You could do if(pos & 1) even++ else odd++ and it would still work. Compact code looks better isn't it?

Regards
Sid